### ISSUE TITLE
Smart entry should display dropdown after the user enters the special characters (e.g. after +, should see a live list of projects to pick from, after @ a date time picker, after # the tags list)

### ISSUE THREAD

**Issue by UrbanCircles** [johannesjo/super-productivity#3544]
![image](https://github.com/user-attachments/assets/42d3131d-53cf-476c-8717-2425e2facda3)

**Comment by bytrangle** [comment]
Typing "#" should already display a dropdown for tag list. So there are only two things left:
- dropdown for projects
- time picker menu
Or are you talking about a different feature?

**Comment by UrbanCircles** [comment]
you're right - this is already partially implemented. only missing the projects and time/ date picker (which ideally should also have common options for tomorrow/ next week/ month, like other editors)

**Comment by bytrangle** [comment]
@johannesjo What date picker do you recommend?
